thirtyfour/src/components/select.rs

```diff
@@ -30,24 +31,30 @@ async fn set_selected(element: &WebElement, select: bool) -> WebDriverResult<()>
     Ok(())
 }
 
+struct Escaped<'a>(&'a str);
+
+impl Display for Escaped<'_> {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        for (i, substring) in self.0.split('\"').enumerate() {
+            if i != 0 {
+                f.write_str(", '\"', ")?
+            }
+            write!(f, "\"{}\"", substring)?;
+        }
+        if self.0.ends_with('\"') {
+            f.write_str(", '\"'")?;
+        }
+        Ok(())
+    }
+}
```

Comment:
This could have just been a function 🤔 ?

Reply:
I don't think so, because I put escaped into a call of `format!()` and I don't think there is a way to access the formatter directly

I kind of use the same patter as any of the `std::str::Escape*`